grpc-ballerina/tests/28_unary_basic_auth_service.bal

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+listener Listener ep28 = new (9118);
+
+@ServiceDescriptor {
+    descriptor: ROOT_DESCRIPTOR_28,
+    descMap: getDescriptorMap28()
+}
+service "HelloWorld28" on ep28 {
+
+    isolated remote function testStringValueReturn(ContextString request) returns string|error {
+        if !request.headers.hasKey(AUTH_HEADER) {
```

Comment:
Shall we integrate one of the listener basic auth handlers here?

Reply:
We only have the LDAP listener basic auth handler. Did you mean to use it here?

Reply:
Oh.. we may need the file user store based basic auth handler. Let's add it and update this test later then. For now, this is okay.

Reply:
Create a separate task to track this https://github.com/ballerina-platform/ballerina-standard-library/issues/1376